### ISSUE TITLE
R14 value should be 2.2k

### ISSUE THREAD

**Issue by xsider** [LibreSolar/mppt-2420-hpx#3]
R14 should be 2.2k instead of 10k.
2.2k is the value expected by the Software.
2.2k should also be used to be save for ADC! 
110k/10k -> Highvoltage over 36.3V could damage microcontroller.

**Comment by martinjaeger** [comment]
Oops, yes, definitely. Thanks for posting.

Looks like I have already fixed this during transition to the MPPT 2420 HC variant. You might want to compare with that schematic in case of any doubts.

Also the internal power supply source switching circuitry in this board (page 3) didn't work properly and has been reworked. If power supply from battery side only is sufficient for you, I would just remove the high-side part around Q9 completely.

**Comment by xsider** [comment]
> Also the internal power supply source switching circuitry in this board (page 3) didn't work properly and has been reworked. If power supply from battery side only is sufficient for you, I would just remove the high-side part around Q9 completely.

That's a good hint. The main cause to build the HPX was the bootstrap-capability from solar power only, which is missing in PWM_2420_LUS. I will consider to adapt the HC circuit for that, or add just a diode from solar panel, because open circuit solar voltage is below 50V.

**Comment by martinjaeger** [comment]
Yeah, it's really useful if you have lithium ion batteries. But that circuit was also quite tricky. The reality was different than theoretical considerations again... the circuit in the develop branch of the HC (v0.2) is finally working as it should :)

But if your voltage is below 50V, a diode should be perfectly fine.
